omnigibson/robots/tiago.py

```diff
@@ -577,6 +710,10 @@ def set_position_orientation(self, position=None, orientation=None):
         if orientation is None:
             orientation = current_orientation
 
+        # Make sure position and orientation are arrays (otherwise set attribute fails)
+        position = np.asarray(position)
```

Comment:
what's the failure case for this? or what type is trying to get passed in here sometimes?

Reply:
Fix `self._world_base_fixed_joint_prim.GetAttribute("physics:localRot0").Set(Gf.Quatf(*orientation[[3, 0, 1, 2]]))`